cirq/__init__.py

```diff
@@ -231,6 +231,7 @@
     ComputeDisplaysResult,
     Linspace,
     ParamResolver,
+    ParamResolverOrSimilarType,
```

Comment:
Why not just ParamResolverType.  I mean "or similar" is kind of vague.  I guess ParamResolverType is also not specific enough.

Reply:
Acknowledged. Yeah, it's not specific enough.